libraries/cyclestreets-core/src/main/res/values-ru/arrays.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string-array name="units">
+    <item>Километры</item>
+    <item>Мили</item>
+  </string-array>
+  <string-array name="unit_values" translatable="false">
+    <item>km</item>
+    <item>miles</item>
+  </string-array>
+
+  <string-array name="speed">
+    <item>Неторопливый 10миля/ч (16км / ч)</item>
+    <item>Запас хода 12миля/ч (20 км / ч)</item>
+    <item>Быстрый 15миля/ч (24км / ч)</item>
+  </string-array>
+  <string-array name="speed_values" translatable="false">
```

Comment:
Okay I will do this. Please write about this on issue page for not repeating this error next time